res/ai/DefaultAIPlayer/TaskBoss.lua

```diff
@@ -101,7 +101,7 @@ function JobCheckCredit:Prepare(pParams)
 	local money = TVT.GetMoney()
 	local creditAvailable = TVT.bo_getCreditAvailable()
 	self.Task.TryToGetCredit = 0
-	if money < 0 and getPlayer().hour > 5 then
+	if money < -30000 and getPlayer().hour > 5 then
```

Comment:
Ich wuerde hier den "small amount" nur akzeptieren, wenn es nicht spaet am Abend ist - bzw man schon den Vortag im Minus war usw. 
Sprich zur Bankrottvermeidung sollte diese Ueberpruefung hier ausgehebelt werden.

Reply:
Das aktuelle KI-Verhalten ist ja, dass bei einem kleinen Minus kurz nach Mitternacht sofort zu Boss gelaufen wird. Solange keine Eigenproduktionen stattfinden, sind Werbevertragsstrafen die einzigen Ausgaben, die zu einem Minus führen können. Wenn die KI es zwsichen 5 und 0 Uhr nicht schafft, Werbeeinnahmen von 30000 zusammenzubekommen, hat sie ganz andere Probleme.

Daher würde ich die Bedingung auch nicht noch komplizierter machen wollen. In den letzten Monaten habe ich bei mir keine KI pleitegehen sehen.

Reply:
OK